src/App.js

```diff
@@ -33,66 +26,24 @@ function App() {
             <ProtectedRoute exact path="/feedback" component={Feedbackscreen} />
             <ProtectedRoute
               exact
-              path="/nutrition/mensa"
-              component={Mensascreen}
+              path="/category/:category"
```

Comment:
Just tested out and works - BUT: also creates a path with "/favorites/1". With sharing links among different users this might cause problems.